### PR TITLE
mimic: mgr/dashboard: Add support for URI encode

### PR DESCRIPTION
**Pull request by tspmelo**
Fixes: http://tracker.ceph.com/issues/24907

### CODE REVIEW

**Comment by smithfarm** [comment]
jenkins re-test this please

**Comment by smithfarm** [comment]
@tspmelo Why is this DNM?

**Comment by tspmelo** [comment]
@smithfarm I was told that https://github.com/ceph/ceph/pull/24183 might be backported to mimic.
If that is the case, this PR would already be included and merging it would created conflicts.

The same applies to https://github.com/ceph/ceph/pull/24478.

File: src/pybind/mgr/dashboard/controllers/__init__.py [hunk 1]
```diff
@@ -16,6 +16,12 @@
 import threading
 import types  # pylint: disable=import-error
 
+if sys.version_info >= (3, 0):
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/controllers/__init__.py]: Funny thing. #24626 contains a better way to import `unquote`

```suggestion
from six.moves.urllib.parse import unquote  # pylint: disable=import-error
```
But I wouldn't change it in Mimic only.

**Comment by LenzGr** [comment]
Waiting for @rjfd to review/approve.

**Comment by yuriw** [comment]
[wip-yuri-testing-2018-10-27-1631-mimic](https://trello.com/c/I9pAs5Dj/454-wip-yuri-testing-2018-10-27-1631-mimic)

**Pull request merged**